src/pages/postgraphile/background_tasks.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+path: /postgraphile/background-tasks/
+title: Background Tasks in PostGraphile 
+---
+
+In your GraphQL server typically most processing will be done in the "frontground", as part of the request-response cycle. HTTP is a blocking protocol: once a client sends a request, it will wait for a reponse. This blocking nature requires the server to be fast at serving responses, to avoid keeping the client waiting for ever.
+
+But sometimes it is impossible to serve the response fast, for example we might need to do some heavy task like creating PDFs, calling an external API that takes a long time to respond, big calculations, or doing flaky tasks that need retries like sending an email. Or you might want to run some code periodically, detached from a user action. This are all good cases for background processing: the code runs outside of main request-response cycle, storing the result somewhere (typically on the database) and (optionally) notifying the client once it has finished.
```

Comment:
```suggestion
Examples of potential background tasks include creating PDFs, sending emails, calling an external API that may be slow, performing big calculations, and the like. Background tasks are also a good way of dealing with tasks where temporary failures are expected, for example you might not be able to send an email immediately - rather than failing the request-response cycle we can just try again (and again, and again) over the coming seconds, minutes, and hours. Another task you might want to solve with background tasks is running code periodically, detached from a user action. These are all good cases for background processing: the code runs outside of main request-response cycle, storing the result somewhere (typically on the database) and (optionally) notifying the client once it has finished.
```